README.md

```diff
@@ -6,14 +6,70 @@
 
 A collection of tools (including data analysis and machine learning applications) that can be used to explore radiation data (temporal and spectral) for external identification of radiation signatures.
 
-## Dependencies
+## Table of Contents
+
+1. [Installation](#Installation)
+
+2. [Usage](#Usage)
+
+## Installation
+
+### Dependencies
 
 * Python 3
 
-Including the following Python packages (found in `requirements.txt`):
+Versions 3.6-3.9 are currently supported by tests. The following Python packages are required to run (found in `requirements.txt`):
 
 * h5py
 * numpy
 * pandas
 * progressbar2
 * scipy
+
+Scripts can be run from the top-level directory:
+
+`python /path/to/RadClass/script`
+
+Or `RadClass` can be installed using pip:
+
+`python -m pip install /path/to/RadClass/. --user`
```

Comment:
How do you run it after it's installed?

Reply:
A script including import statements (i.e. `import RadClass` or `from RadClass import H0`) and then execute the script with `python /path/to/script.py`.

Reply:
This is perhaps a bit of a confusing juxtaposition, then.  The first suggestion is to just run scripts in place, suggesting that the way to use it is to run a script.  Then you suggest the alternative of installing it, but there is no equivalent for now to run it. (Not sure if I'm making sense....)

Reply:
Ah yes, you do make sense. Ultimately, RadClass requires a user script to run, with the expectation of the user importing RadClass module(s) to their program. I think there's just some formalism to clean up, but we should debug offline to make sure I understand.